subprojects/tracy.wrap

```diff
@@ -0,0 +1,3 @@
+[wrap-git]
+url = https://github.com/wolfpld/tracy.git
```

Comment:
I also tried to build this, and this submodule is not cloned unless explicitly enabled, which is good.